src/PlotlyJS.jl

```diff
@@ -163,6 +163,19 @@ function __init__()
             dataset(::Type{DataFrames.DataFrame}, name::String) = DataFrames.DataFrame(dataset(CSV.File, name))
         end
     end
+
+    wait(kaleido_task)
+
+    if ccall(:jl_generating_output, Cint, ()) == 1
+        # ensure precompilation of packages depending on PlotlyJS finishes
+        if isdefined(P, :proc)
+            close(P.proc)
+            close(P.stdin)
+            close(P.stdout)
+            close(P.stderr)
+            sleep(0.1) # avoid a "waiting for IO to finish" warning
```

Comment:
A forgotten `wait(P)` is why you are getting the warning because that is a bug. Adding `sleep` as a workaround just makes your code more unreliable.

Reply:
Thanks! https://github.com/JuliaPlots/PlotlyJS.jl/pull/469/commits/d13cf03c36c789ca1d353709b47cb09b9192e5f7